provider/datagen/src/transform/cldr/units/helpers.rs

```diff
@@ -281,9 +281,7 @@ pub fn process_constants<'a>(
 /// - " 1.5 E -2 " is converted to 15/1000
 /// - " 1.5 E - 2" is an invalid scientific notation number
 /// - "1.5E-2.5" is an invalid scientific notation number
-pub fn convert_scientific_notation_to_fraction(
-    number: &str,
-) -> Result<GenericFraction<BigUint>, DataError> {
+pub fn convert_scientific_notation_to_fraction(number: &str) -> Result<Ratio<BigInt>, DataError> {
```

Comment:
why not using [`BigRational`](https://docs.rs/num/latest/num/type.BigRational.html)

Reply:
There's no need to add a dep on the `num` metacrate for this, and BigRational is just a typedef

Reply:
We're trying to keep dependencies low; I'm fine pulling in more `num` crates but we don't have to pull in _all_ of them.

Reply:
Okay, got it